DEPLOYMENT_FAQ.md

```diff
@@ -0,0 +1,78 @@
+## Deployment Frequently Asked Questions (FAQ)
```

Comment:
We have a `documentation` folder, is this something we should place it in there?